ert3/evaluator/_evaluator.py

```diff
@@ -123,7 +123,7 @@ def _fetch_results(ee_config, ensemble, stages_config):
 def _run(ensemble_evaluator):
     monitor = ensemble_evaluator.run()
     for event in monitor.track():
-        if event.data is not None and event.data.get("status") == "Stopped":
```

Comment:
Shouldn't the failure be "handled" here? Printing something to the console or some such?

Reply:
I think we said that logging would be the appropriate response for now. At that point it should have been logged already. 

Also: the status is set to Failed, should that not be reported  by the GUI/CLI somehow?

Reply:
You're right. Could you create a new issue?